Plugin/src/SofaPython3/DataHelper.cpp

```diff
@@ -249,10 +249,17 @@ py::slice toSlice(const py::object& o)
 
 std::map<void*, std::pair<int, py::array>>& getObjectCache()
 {
-    static std::map<void*, std::pair<int, py::array>> s_objectcache {} ;
+    static std::map<void*, std::pair<int, py::array>>  s_objectcache{};
     return s_objectcache;
 }
 
+
+void clearCache()
+{
+    msg_info("SofaPython3") << "Clearing Sofa.Core cache...";
```

Comment:
Why Sofa.Core whereas this code is not in Sofa.Core?

Reply:
Yes, but this cache is only used in Sofa.Core, in the data binding.